src/components/TimeBasedLineChart.js

```diff
@@ -1,32 +1,245 @@
 const React = require('react');
-const ReactDom = require('react-dom');
 const Radium = require('radium');
 
 const d3 = require('d3');
-const { isEqual } = require('lodash');
 const moment = require('moment');
 const numeral = require('numeral');
 
+const BreakPointGroup = require('./d3/BreakPointGroup');
+const GridLinesGroup = require('./d3/GridLinesGroup');
+const CirclesGroup = require('./d3/CirclesGroup');
+const LineGroup = require('./d3/LineGroup');
+const ShadedRectangleGroup = require('./d3/ShadedRectangleGroup');
+const SlicesGroup = require('./d3/SlicesGroup');
+const TimeXAxisGroup = require('./d3/TimeXAxisGroup');
+const AxisGroup = require('./d3/AxisGroup');
+
 const StyleConstants = require('../constants/Style');
 
-class TimeBasedLineChart extends React.Component {
-  constructor (props) {
-    super(props);
+const ChartUtils = require('../utils/Chart');
+
+const styles = {
+  // NOTE: D3 doesn't like camel cased key names for
+  // styles.  Because of this styles in this file may
+  // resemble the following.  'font-size'  DO NOT
+  // change or chart styles will break.
+
+  // Component
+  component: {
+    fontFamily: StyleConstants.FontFamily,
+    position: 'relative',
+    boxSizing: 'content-box',
+    display: 'inline-block'
+  },
+
+  // Chart
+  breakPointLabel: {
+    fill: StyleConstants.Colors.ASH,
+    'font-family': StyleConstants.Fonts.REGULAR,
```

Comment:
Yes, I noted in the comment above the objects.  D3 cas to have them as `font-family` or things don't work.